gmso/core/potential.py

```diff
@@ -47,28 +44,35 @@ def __init__(self,
                  template=False,
                  topology=None
                  ):
+        self._name = name
+
+        if expression is None:
```

Comment:
Yeah. This might have to do with a lot of our core classes constructor arguments. While it make life easier to toy around and write tests to just instantiate a class, some keyword arguments could/should be changed to positional. However, I would argue that this is not in the scope of this PR.

Reply:
We definitely need to rethink this default expression for the potential, but yeah, I agree it's kinda belong to its own PR